### DISCUSSION TITLE
use ctrl for more

### DISCUSSION THREAD

**Discussion by 4a3** [FreeCAD/FreeCAD#11898] [Discussions that will turn in to issues but need refining]
you can save a lot of time if you have a quick crop function. this is such a frequent operation that I have already set it to ctrl
https://github.com/FreeCAD/FreeCAD/assets/829542/aee58062-edee-4526-aa5c-7a7d6d55bd0a

**Comment by pierreporte** [comment]
You should create a feature request: https://github.com/FreeCAD/FreeCAD/issues/new/choose
